spec/cohortManager.spec.js

```diff
@@ -0,0 +1,218 @@
+const { CohortManager, Cohort, Student } = require('./../src/cohortManager.js')
```

Comment:
youd want some tests for Cohort and Student class, even just to make sure that the constructors save hte right data when an obj is created.